policyengine_us/parameters/gov/states/mt/tax/income/credits/ctc/reduction/threshold.yaml

```diff
@@ -0,0 +1,13 @@
+description: Montana reduces the child tax credit for filers with income above this income threshold.
+values:
+  2021-01-01: 0
+  2023-01-01: 50_000
+metadata:
+  units: currency-USD
+  period: year
+  label: Montana child tax credit reduction threshold
+  reference:
+  # The Montana Child Tax Credit established in the 2023 House Bill No. 268
+  # not existent before
+    - title: 2023 Montana Legislature, House Bill No. 268, Section 1, 6
```

Comment:
exact section of the law

Reply:
check all yaml files

Reply:
Thank you for the comments. I'm a bit confused about what do you mean by exact section of the law? Do you mean legal codes related to this threshold other than 2023 Montana Legislature, House Bill No. 268, Section 1, 6? Thank you!